lib/node/failure_detector.mli

```diff
@@ -1,7 +1,7 @@
 open Types
 open Common
 
-(** Messages sent by the failure detector protocol *)
```

Comment:
Keep it as a documentation comment for hovering :smile:

Reply:
With two stars?